### PR TITLE
WIP: dapsi2blogpost

### PR DESCRIPTION
**Pull request by Yettte**
Blogpost for dapsi phase 2 wrap up:

- [x] upload comparison table (spreadsheet) to https://github.com/deltachat/eppdperf
- [x] create screenshot of "condensed" comparison table

### CODE REVIEW

**Comment by missytake** [comment]
Hm, right now the page isn't displayed - you need to delete the first (empty) line, and rename the file, so the `_` is a `-`

Apparently it doesn't work with _ in the filename :/

**Comment by Yettte** [comment]
> Hm, right now the page isn't displayed - you need to delete the first (empty) line, and rename the file, so the `_` is a `-`
> 
> Apparently it doesn't work with _ in the filename :/

thanks nami! I changed the file name, should work now

**Comment by missytake** [comment]
Cool :) the spaces in the links (between the `[]` and the `()`) have to go away, too, but some of the links aren't complete yet anyway, right?

File: _posts/2022-01-16-dapsi2blogpost.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+---
+title: Which E-Mail Providers work well with Delta Chat? Quite some :) 
+author: missytake, jette, holga
+image: ../assets/blog/eppd-criteria-final.jpg
+---
+
+Coming to an end with our EU [NGI](https://www.ngi.eu/) funded E-mail Provider Comparison Directory (EPPD) project, we gathered some interesting results about providers' performances and how they can influence the use of our Delta Chat app suite.  
+
+The [DAPSI](https://dapsi.ngi.eu/hall-of-fame/eppd/) funding did not only result in an email provider comparison directory but also helped to improve Delta Chat's onboarding and usability! We released [quota and connectivity] (https://delta.chat/en/2021-08-24-updates) and the [account switcher] (https://delta.chat/en/2021-11-17-releases) which we are quite happy with.
```
Comment (missytake) [inline _posts/2022-01-16-dapsi2blogpost.md]:
```suggestion
The [DAPSI](https://dapsi.ngi.eu/hall-of-fame/eppd/) funding did not only result in an email provider comparison directory but also helped to improve Delta Chat's onboarding and usability! We released [quota and connectivity](https://delta.chat/en/2021-08-24-updates) and the [account switcher](https://delta.chat/en/2021-11-17-releases) which we are quite happy with.
```

like this :)

**Comment by Yettte** [comment]
> Cool :) the spaces in the links (between the `[]` and the `()`) have to go away, too, but some of the links aren't complete yet anyway, right?

Exactly, I put them only as placeholders

File: _posts/2022-01-16-dapsi2blogpost.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+author: missytake, jette, holga
+image: ../assets/blog/eppd-criteria-final.jpg
+---
+
+In the last couple of months we engaged with comparing around 20 international e-mail providers, based on our [earlier research into comparison criteria](https://delta.chat/en/2021-09-11-eppd-provider-criteria). These efforts were part of our [EPPD](https://dapsi.ngi.eu/hall-of-fame/eppd/) project funded by the EU. 
+
+<a href="../assets/blog/eppd-comparison-final.pdf">
+    <img src="../assets/blog/2022-01-19-eppd-table.png"
+         width="260" style="border-width: 1px; border-color: grey; border-style: solid;"/><br>
+    <b>Download</b>provider comparison table.
```
Comment (missytake) [inline _posts/2022-01-16-dapsi2blogpost.md]:
```suggestion
    <b>Download</b> provider comparison table.
```

one small typo :)

File: _posts/2022-01-16-dapsi2blogpost.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+title: Which e-mail providers work well with Delta Chat? Quite some :) 
+author: missytake, jette, holga
+image: ../assets/blog/eppd-criteria-final.jpg
+---
+
+In the last couple of months we engaged with comparing around 20 international e-mail providers, based on our [earlier research into comparison criteria](https://delta.chat/en/2021-09-11-eppd-provider-criteria). These efforts were part of our [EPPD](https://dapsi.ngi.eu/hall-of-fame/eppd/) project funded by the EU. 
+
+<a href="../assets/blog/eppd-comparison-final.pdf">
+    <img src="../assets/blog/2022-01-19-eppd-table.png"
+         width="260" style="border-width: 1px; border-color: grey; border-style: solid;"/><br>
```
Comment (missytake) [inline _posts/2022-01-16-dapsi2blogpost.md]:
```suggestion
         style="border-width: 1px; border-color: grey; border-style: solid;"/><br>
```

260 width is really small; if we leave it out, it looks quite good :)

**Review by missytake — CHANGES_REQUESTED** [review]
2 small things :)

**Review by missytake — APPROVED** [review]

**Pull request merged**